### PR TITLE
My Personal Website - HTML only 

### PR DESCRIPTION
**Pull request by PhilippAubert**

### CODE REVIEW

**Comment by johdyn** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            <nav>
+        
+                <ul>                
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#">Career</a></li>
+                </ul>
+        
+            </nav>
+            
+            <figure>
```
Comment (namirsab) [inline index.html:27]: Amazing, using `figure`! very good!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+        <main>
+
+        <section id="statement"> 
+
+          <h2>Interests</h2>
+
+          <article> 
+            <h2>Abstract Painting</h2>
+            <p>Challening space with the line</p>
```
Comment (namirsab) [inline index.html]: I think you want to write here `Challenging`?

File: markup-assign.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
```
Comment (namirsab) [inline markup-assign.html]: You have 2 files with the same content. Is this intended?

**Review by mpagels — APPROVED** [review]
What does the markup-assign.html do? Seems like a copy. If not needed you can delete it before merge, to avoid data "noise" in the project. ;-)

Very good Job Philipp!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="utf-8">
+        <title>Assignment HTML</title>
```
Comment (namirsab) [inline index.html]: Even though this is an assignemnt, think of it as your personal project. give it a title that you'd like to see in a the browser tab 😉

**Review by namirsab — CHANGES_REQUESTED** [review]
Hey @PhilippAubert really good job! I love the content, I had lots of fun reading it! (and i'll definetely check your soundcloud haha)

I've added some little requests, please take a look to them. Just some small details.

**Review by mpagels — APPROVED** [review]

File: markup-assign.html (hunk 3 above)
Comment (PhilippAubert) [reply markup-assign.html]: I still do. Of course, index.html is the proper one. 

This happened before I did the git push - I removed the file and even deleted the branch, but it didn't update so I must have missed something at one point. I'll take care of that right now. :blush:

File: index.html (hunk 1 above)
Comment (PhilippAubert) [reply index.html:27]: Thank you :blush:

File: index.html (hunk 4 above)
Comment (PhilippAubert) [reply index.html]: You're absolutely correct. :satisfied:

**Review by PhilippAubert — COMMENTED** [review]
Thanks for the input, Namir and Martin. You guys rock :blush:

File: index.html (hunk 2 above)
Comment (PhilippAubert) [reply index.html]: I did. What a blunder :satisfied:

**Pull request merged**